traffic_ops/v4-client/division.go

```diff
@@ -52,6 +52,17 @@ func (to *Session) GetDivisions(header http.Header) ([]tc.Division, toclientlib.
 	return data.Response, reqInf, err
 }
 
+// GetDivisions with Params returns all Divisions with matching params in Traffic Ops
+func (to *Session) GetDivisionsWithParams(header http.Header, params url.Values) ([]tc.Division, toclientlib.ReqInf, error) {
```

Comment:
For client functions with both `url.Values` and `http.Header` in the function signature, the convention is for `http.Header` to still be the last function parameter. Examples:

* https://github.com/apache/trafficcontrol/blob/d3bf7cc76c55d8305a8017eb607a08402e990d17/traffic_ops/v4-client/asn.go#L48
* https://github.com/apache/trafficcontrol/blob/d3bf7cc76c55d8305a8017eb607a08402e990d17/traffic_ops/v4-client/cachegroup.go#L83
* https://github.com/apache/trafficcontrol/blob/d3bf7cc76c55d8305a8017eb607a08402e990d17/traffic_ops/v4-client/cachegroup_parameters.go#L33
* https://github.com/apache/trafficcontrol/blob/d3bf7cc76c55d8305a8017eb607a08402e990d17/traffic_ops/v4-client/deliveryserviceserver.go#L69
* https://github.com/apache/trafficcontrol/blob/d3bf7cc76c55d8305a8017eb607a08402e990d17/traffic_ops/v4-client/federation_resolver.go#L28
* https://github.com/apache/trafficcontrol/blob/d3bf7cc76c55d8305a8017eb607a08402e990d17/traffic_ops/v4-client/phys_location.go#L60
* https://github.com/apache/trafficcontrol/blob/d3bf7cc76c55d8305a8017eb607a08402e990d17/traffic_ops/v4-client/profile_parameter.go#L52
* https://github.com/apache/trafficcontrol/blob/d3bf7cc76c55d8305a8017eb607a08402e990d17/traffic_ops/v4-client/role.go#L64
* https://github.com/apache/trafficcontrol/blob/d3bf7cc76c55d8305a8017eb607a08402e990d17/traffic_ops/v4-client/servercheck.go#L43
* https://github.com/apache/trafficcontrol/blob/d3bf7cc76c55d8305a8017eb607a08402e990d17/traffic_ops/v4-client/server.go#L129
* https://github.com/apache/trafficcontrol/blob/d3bf7cc76c55d8305a8017eb607a08402e990d17/traffic_ops/v4-client/serviceCategory.go#L50

Reply:
removed GetDivisionsWithParams and replaced with GetDivisions